src/suricata.c

```diff
@@ -648,12 +647,6 @@ static void PrintUsage(const char *progname)
 #ifdef HAVE_NETMAP
     printf("\t--netmap[=<dev>]                     : run in netmap mode, no value select interfaces from suricata.yaml\n");
 #endif
-#ifdef HAVE_PFRING
```

Comment:
can we do this in a way that remains backwards compatible? E.g. convert these options internally to the correct ConfSets and do a validation as soon as the plugin is loaded?

Reply:
Yeah, we can do something. Essentially give Suricata special knowledge of this plugin.